packages/lit-dev-content/samples/examples/motion-simple/project.json

```diff
@@ -0,0 +1,15 @@
+{
+  "extends": "/samples/base.json",
+  "title": "simple",
+  "description": "Simple `animate()` directive example.",
+  "section": "Motion",
+  "files": {
+    "my-element.ts": {},
+    "index.html": {},
+    "package.json": {
+      "content": "{\n  \"dependencies\": {\n    \"lit\": \"^2.0.0\",\n    \"@lit/reactive-element\": \"^1.0.0\",\n    \"lit-element\": \"^3.0.0\",\n    \"lit-html\": \"^2.0.0\"\n,\n    \"@lit-labs/motion\": \"^1.0.0\",  }\n}",
```

Comment:
Should only need the `lit` and `motion` deps here?

Reply:
Done.